backend/config/database.yml

```
@@ -0,0 +1,16 @@
+  host: db
+  username: <%= ENV.fetch("POSTGRES_USER") { "user" } %>
+  password: <%= ENV.fetch("POSTGRES_PASSWORD") { "postgres" } %>
+  port: 5432
+
+development:
+  <<: *default
+  database: myapp_development
+
+test:
+  <<: *default
+  database: myapp_test
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)